src/test/test_schema.py

```diff
@@ -27,11 +27,17 @@
 from ssvc.decision_points.critical_software import CRITICAL_SOFTWARE_1  # noqa
 from ssvc.decision_points.high_value_asset import HIGH_VALUE_ASSET_1  # noqa
 from ssvc.decision_points.in_kev import IN_KEV_1
-from ssvc.dp_groups.cvss.collections import CVSSv1, CVSSv2, CVSSv3, CVSSv4  # noqa
+from ssvc.dp_groups.cvss.collections import (
+    CVSSv1,
+    CVSSv2,
+    CVSSv3,
+    CVSSv4,
+)  # noqa
 
 # importing these causes the decision points to register themselves
 from ssvc.dp_groups.ssvc.collections import SSVCv1, SSVCv2, SSVCv2_1  # noqa
 
+
 def retrieve_local(uri):
     fileuri = uri.replace("https://certcc.github.io/SSVC", os.getcwd())
```

Comment:
If we are fixing the tests, we can make this an environment variable so local mkdocs/docker can do this test without the remote dependency?

Reply:
I mean make the var `https://certcc.github.io/SSVC` from an environment variable, that is set when running tests.

Reply:
I'll address this thread in #669